Automattic/Gravatar-SDK-iOS
# Set icon size programmatically so that we don't need to manually adjust pdf icon sizes

Follow up work for https://github.com/Automattic/Gravatar-SDK-iOS/pull/189
Closes: #197

### Description

We had to manually adjust icon size according to the designs before. After this PR we won't need that.

By default the UIButton's UIImageView has an intrinsic size equal to its image's size. But we don't always want to rely on icon's size, especially if it's a vector image that should be able to be rendered in any size. So we explicitly set UIImageView size. I think there should be a better way to do this but 🤷‍♀️.

### Testing Steps

- Change `accountIconLength` to any size in the code and check if the icon size adjusts fine and the icon quality is good.
